packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -162,67 +176,48 @@ class DDSTabsExtended extends StableSelectorMixin(LitElement) {
     return tabItems;
   }
 
-  updated() {
+  updated(_changedProperties) {
+    const { _isMobileVersion } = this;
     this._isLTR = window.getComputedStyle(this).direction === 'ltr';
     this._activeTabIndex = parseInt(this._activeTab, 10);
 
-    this._tabItems.map((tab, index) => {
+    this._tabItems.forEach((tab, index) => {
       (tab as DDSTab).selected = index === this._activeTabIndex;
       (tab as DDSTab).setIndex(index);
-      const navLink = this.shadowRoot!.querySelectorAll(
-        `.${prefix}--tabs__nav-link`
-      )[index];
-      const navText = navLink!.querySelector('div p');
-      if (navText!.scrollHeight > navText!.clientHeight) {
-        const label = (tab as DDSTab).getAttribute('label');
-        if (label) {
-          navLink!.setAttribute('aria-label', label);
-          navLink!.setAttribute('hasTooltip', label);
+
+      if (_changedProperties.has('_isMobileVersion')) {
+        // Attach accordion behavior on mobile.
+        if (_isMobileVersion) {
+          tab.addEventListener('click', (e) => this._handleClick(index, e));
```

Comment:
It's a small thing, and admittedly an edge case, but I'm seeing that there is potential for this event handler to be applied multiple times if you cross back and forth over the breakpoint many times.

Reply:
Good catch! Made some changes so that we attach a named function, which the browser shouldn't re-attach on subsequent `addEventListener` calls.

For posterity, from https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener:
> Note: If a particular anonymous function is in the list of event listeners registered for a certain target, and then later in the code, an identical anonymous function is given in an addEventListener call, the second function will also be added to the list of event listeners for that target.

and

> If the function or object is already in the list of event listeners for this target, the function or object is not added a second time.